File: libs/eave-stdlib-py/src/eave/stdlib/core_api/models/slack.py
import uuid
from eave.stdlib.core_api.models import BaseInputModel, BaseResponseModel


import pydantic


class SlackInstallation(BaseResponseModel):
    id: uuid.UUID
    team_id: uuid.UUID
    """eave TeamOrm model id"""
    slack_team_id: str
    bot_token: str

class SlackInstallationPeek(BaseResponseModel):
    id: uuid.UUID
    team_id: uuid.UUID
    """eave TeamOrm model id"""
    slack_team_id: str


class SlackInstallationInput(BaseInputModel):
    slack_team_id: str

File: libs/eave-stdlib-ts/src/core-api/models/slack.ts
export type SlackInstallationInput = {
  slack_team_id: string;
};

export type SlackInstallation = {
  id: string;
  /** eave TeamOrm model id */
  team_id: string;
  slack_team_id: string;
  bot_token: string;
  bot_id: string;
  bot_user_id: string | null;
};
